### ISSUE TITLE
diagonal gates

### ISSUE THREAD

**Issue by vlzal** [Fuzss/diagonalfences#23]
### Mod Loader (Optional)

Fabric

### Minecraft Version(s) (Optional)

1.20.1

### Mod Version(s) (Optional)

v8.1.3

### Suggestion (Required)

Make the gates diagonal too pls

**Comment by JustinTheBob** [comment]
Yes pleaseeeee that would be amazing! Please and thank you for an amazing mod.
